sdk/textanalytics/Azure.AI.TextAnalytics/src/PageableOperationHelpers.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal static class PageableOperationHelpers
```

Comment:
I wanted to use or modify `PageableHelpers` to avoid repeating logic, but because I always calculate the first page I ended up writing a new class.
Is there a better way to do this?

Reply:
Did you try using the https://github.com/Azure/autorest.csharp/blob/feature/v3/src/assets/Generator.Shared/PageableHelpers.cs#L21

but passing a func that always returns the firstPageResults.


`PageableHelpers.CreateAsyncEnumerable(_ => Task.FromResult(firstPageResults), ...)`?

Reply:
so simple... I did not think of that :) 
Thank you!!!